packages/@react-spectrum/dropzone/src/DropZone.tsx

```diff
@@ -0,0 +1,85 @@
+/*
+ * Copyright 2023 Adobe. All rights reserved.
+ * This file is licensed to you under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License. You may obtain a copy
+ * of the License at http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under
+ * the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
+ * OF ANY KIND, either express or implied. See the License for the specific language
+ * governing permissions and limitations under the License.
+ */
+
+import {AriaLabelingProps, DOMProps, FocusableRefValue, StyleProps} from '@react-types/shared';
+import {classNames, createFocusableRef, SlotProvider, useStyleProps} from '@react-spectrum/utils';
+import {DropOptions, mergeProps} from 'react-aria';
+import {filterDOMProps} from '@react-aria/utils';
+import {DropZone as RACDropZone} from 'react-aria-components';
+import React, {ReactNode, Ref, useImperativeHandle, useRef} from 'react';
+import styles from '@adobe/spectrum-css-temp/components/dropzone/vars.css';
+export interface SpectrumDropZoneProps extends Omit<DropOptions, 'getDropOperationForPoint' | 'ref' | 'hasDropButton'>, DOMProps, StyleProps, AriaLabelingProps {
+  children: ReactNode,
+  isFilled: boolean, 
+  bannerMessage: string
+}
+
+export interface DropZoneRef extends FocusableRefValue<HTMLInputElement, HTMLDivElement> {
+  getInputElement(): HTMLInputElement | HTMLDivElement | null
+}
+
+function DropZone(props: SpectrumDropZoneProps, ref: Ref<DropZoneRef>) {
+  let {children, isFilled, bannerMessage, ...otherProps} = props;
+  let {styleProps} = useStyleProps(otherProps);
+  let domRef = useRef<HTMLDivElement>(null);
+  let inputRef = useRef<HTMLInputElement>(null);
+
+  let domProps = filterDOMProps(props, {labelable: true});
+  delete domProps.id;
+
+  // Expose imperative interface for ref
+  useImperativeHandle(ref, () => ({
+    ...createFocusableRef(domRef, inputRef),
+    getInputElement() {
+      return inputRef.current;
+    }
+  }));
+
+  return (
+    <RACDropZone
+      // {...styleProps} working on getting this to work...
+      {...mergeProps(otherProps, domProps)}
+      className={
+      classNames(
+        styles,
+        'spectrum-Dropzone',
+        styleProps.className
+      )} 
+      ref={domRef}>
+      {({isDropTarget}) => (
+        <div>
+          {isFilled && isDropTarget && 
+            <div
+              className={
+                classNames(
+                  styles,
+                  'spectrum-Dropzone-banner',
+                  styleProps.className
+                )
+              }>
+              {bannerMessage ? bannerMessage : 'Drop file to replace'}
+            </div>}
+          <SlotProvider
+            slots={{
+              illustration: {UNSAFE_className: classNames(styles, 'spectrum-IllustratedMessage')}, 
```

Comment:
This should be a class name specific to dropzone, like `spectrum-Dropzone--illustratedMessage`, then we can style that instead of the `.spectrum-Dropzone[data-focus-visible] svg` selector currently used.